cogboard-webapp/src/components/styled/index.js

```diff
@@ -128,12 +128,29 @@ export const StyledFormControl = styled(FormControl)`
   max-width: 300px;
 `;
 
-export const StyledPopoverText = styled(Typography)`
+export const StyledPopoverText = styled('div')`
```

Comment:
When you create a styled component for html selector you can simply write
```suggestion
export const StyledPopoverText = styled.div`
```

Reply:
Fixed in commit 297f448